src/Relude/Functor/Reexport.hs

```diff
@@ -1,13 +1,14 @@
 {-# LANGUAGE Safe #-}
 
-{-
+{- |
 Copyright: (c) 2016 Stephen Diehl
            (c) 20016-2018 Serokell
            (c) 2018 Kowainik
-License: MIT
--}
+License:    MIT
+Maintainer: Kowainik <xrom.xkov@gmail.com>
 
--- | This module reexports functionality regarding 'Functor' type class.
+This module reexports functionality regarding 'Functor' type class.
```

Comment:
I noticed that short style description is looking better. Probably we should consider replacing with something like this:
_ Functionality for 'Functor' type class._
What do you like more?